matsuoharuki/pbl2_group2
# #24 Feature/saveReview

Description: 
- Added a function saving review data to the database

To test this pull request:
- From local, run this command
```
docker-compose up
```
- Send the POST request such as
```
 http://localhost:5000/review/register/ 
```
- Below is sample data.
```
{
    "spotId" : "spotXXXX",
    "comment" : "commentXXX", 
    "score" : 100000, 
    "userId" : "userXXX"
}
```
- when you send request, you can see your sample data in the database

Success condition:
```
{
    "success": true,
    "userId": "userXXX",
    "comment": "commentXXX",
    "score": 100000
}
```
Exception: 
- When you post the data whose spotId is not exist, you will get error message
```
[ReviewSQL] error: There is no spot at: SpotId
```
